www/plugins/gatsby-plugin-i18n/gatsby-node.js

```diff
@@ -0,0 +1,15 @@
+exports.onCreatePage = ({ page, actions }, pluginOptions) => {
+  const { createPage } = actions
+  const { languages } = pluginOptions
+
```

Comment:
I'm trying to find a better explanation than "we always did it like that" for telling you to also add a `deletePage` before that, like here:
https://github.com/gatsbyjs/gatsby/blob/master/examples/using-i18n/gatsby-node.js#L12-L14

If I can get `www` running I'll inspect if we indeed have double the pages for src/pages files.

Reply:
Doesn't seem like Gatsby has problems with it though.

![image](https://user-images.githubusercontent.com/16143594/75170376-4e2c7200-572a-11ea-9f3a-4b141a376a25.png)

Reply:
English isn't counted among the languages in the options so it doesn't end up creating a new page for English.

Reply:
Ah, yeah, makes total sense 👍 Thought english would also be in the json file.

Reply:
Definitely want to change that at some point but right now this is easier. 🤷‍♀️English is also a special case because the docs are sourced from the filesystem and not another repo.